.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
```

Comment:
I thought about this but figured it was fine since this is a config file for vcscode, and vscode supports comments. But happy to change if it bothers you.